test/session_test.rb

```diff
@@ -519,7 +519,7 @@ def make_sorted_params(params)
 
   def generate_signature(params)
     params = make_sorted_params(params) if params.is_a?(Hash)
-    OpenSSL::HMAC.hexdigest(OpenSSL::Digest::SHA256.new, ShopifyAPI::Session.secret, params)
+    OpenSSL::HMAC.hexdigest(OpenSSL::Digest.new('SHA256'), ShopifyAPI::Session.secret, params)
```

Comment:
I would be cautious of how this might impact different Ruby versions.

Reply:
We run the tests for every ruby version between 2.4 and 2.7, and the CI build worked. Is there anything else in particular we should look for?

Reply:
Ok, then it should be fine. I'm just cautious of OpenSSL related things being fragile, based on past bad experience!